### PR TITLE
Add method to return list of broker that supports account manager

### PR DESCRIPTION
**Pull request by rpdome**
Related PR: https://github.com/AzureAD/ad-accounts-for-android/pull/2339

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt [hunk 1]
```diff
@@ -160,6 +169,18 @@ data class BrokerData(val packageName : String,
             AuthenticationConstants.Broker.LTW_APP_SHA512_DEBUG_SIGNATURE
         )
 
+        @JvmStatic
+        val accountManagerBrokers: Set<String> =
+            Collections.unmodifiableSet(object : HashSet<String>() {
```
Comment (shahzaibj) [inline common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt:126]: I believe this creates a new object on each invokation of the method? Could the (unmodifiable) list be declared as a constant in this file and then just return that constant?

**Review by shahzaibj — APPROVED** [review]

File: common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt (hunk 1 above)
Comment (rpdome) [reply common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt:126]: This is a static constant (val), not a method.

const in kotlin means compile-time constant. val = runtime constant.

Comment (shahzaibj) [reply common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt:126]: Can we move it and all constants to the top of the file like we do in Java files (for easy discoverability)?

**Comment by shahzaibj** [comment]
Add changelog

File: common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt [hunk 2]
```diff
@@ -67,45 +67,6 @@ data class BrokerData(val packageName : String,
          */
         val sShouldTrustDebugBrokers = BuildConfig.DEBUG
 
-        /**
-         * Returns the list of known broker apps (which SDK should make requests to).
-         * see [sShouldTrustDebugBrokers] for more info regarding testing.
-         **/
-        fun getKnownBrokerApps() : Set<BrokerData> {
```
Comment (rpdome) [inline common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt:74]: move methods below constants/variables as suggested by shahzaib.
no other changes.

**Review by iamgusain — APPROVED** [review]
:shipit:

File: common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt [hunk 3]
```diff
@@ -160,6 +121,18 @@ data class BrokerData(val packageName : String,
...
 
+        @JvmStatic
+        val accountManagerBrokers: Set<String> =
+            Collections.unmodifiableSet(object : HashSet<String>() {
+                init {
+                    add(AuthenticationConstants.Broker.AZURE_AUTHENTICATOR_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.BROKER_HOST_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.MOCK_AUTH_APP_PACKAGE_NAME)
+                    add(AuthenticationConstants.Broker.MOCK_CP_PACKAGE_NAME)
```
Comment (iamgusain) [inline common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt:132]: Should these be added behind the check sShouldTrustDebugBrokers

Comment (rpdome) [reply common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt:132]: It already is (but as a set of BrokerData, not just names like this one object).

**Pull request merged**